tox.ini

```diff
@@ -3,13 +3,10 @@
 ################################################################################
 
 [constants]
-source_locations =
-    py_trees
-    tests
-#    doc/examples
+source_locations = py_trees tests doc/examples
```

Comment:
multiline stopped working in newer tox versions, cool and good

https://github.com/tox-dev/tox/issues/1933